README.md

```diff
@@ -1,31 +1,175 @@
-# schema-ui
+# reproschema-ui
 
 [![Build Status](https://travis-ci.org/ReproNim/schema-ui.svg?branch=master)](https://travis-ci.org/ReproNim/schema-ui)
 
+This repository contains the code for the user-interface for the [ReproSchema](https://github.com/ReproNim/reproschema).
 
-## Build Setup
+See it in action [HERE](https://schema.repronim.org/ui/)
 
+<!-- TOC -->
+
+- [reproschema-ui](#reproschema-ui)
+  - [List of supported input types](#list-of-supported-input-types)
+    - [Text inputs](#text-inputs)
+    - [Numerical values](#numerical-values)
+    - [Audio inputs](#audio-inputs)
+    - [Time and dates inputs](#time-and-dates-inputs)
+    - [Multiple options inputs](#multiple-options-inputs)
+    - [Others](#others)
+  - [App architecture](#app-architecture)
+  - [Reference material](#reference-material)
+  - [Serve the app on your computer](#serve-the-app-on-your-computer)
+    - [Install the dependencies](#install-the-dependencies)
+    - [Serve the app locally](#serve-the-app-locally)
+    - [Build for production with minification](#build-for-production-with-minification)
+    - [Build for production and view the bundle analyzer report](#build-for-production-and-view-the-bundle-analyzer-report)
+    - [Run unit tests](#run-unit-tests)
+    - [Run e2e tests](#run-e2e-tests)
+    - [Run all tests](#run-all-tests)
+
+<!-- /TOC -->
+
+## List of supported input types
```

Comment:
I suspect that a simplified version of this section could be used in the reproschema doc.